src/analysis/souffle/examples/multimic/BUILD

```diff
@@ -15,19 +15,77 @@
 #
 #-------------------------------------------------------------------------------
 
-licenses(["notice"])
+load("//build_defs:raksha_policy_verifier.bzl", "raksha_policy_verifier_library")
 
-load("//build_defs:raksha.bzl", "policy_check")
+package(
+    default_visibility = ["//src:__subpackages__"],
+    features = ["layering_check"],
+    licenses = ["notice"],
+)
+
+# policy_check(
+#     name = "check_multimic_userc_tag_fail",
+#     auth_logic = "multimic.auth",
+#     dataflow_graph = "multimic.arcs",
+#     expect_failure = True,
+# )
 
-policy_check(
+# policy_check(
+#     name = "check_multimic_pass",
+#     auth_logic = "multimic_no_userc_tag.auth",
+#     dataflow_graph = "multimic.arcs",
+# )
+
```

Comment:
Add a comment to this file describing how you generated the datalog files use here?